### PR TITLE
Assignment

### PR DESCRIPTION
**Pull request by daquinons**

### CODE REVIEW

File: assignments/callbacks.js [hunk 1]
```diff
@@ -27,23 +27,29 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
```
Comment (GianBarGian) [inline assignments/callbacks.js:35]: You are not invoking your functions anywhere. You could want to write a separate logger function and invoke the various functions passing that as the second argument.

File: assignments/closure.js [hunk 2]
```diff
@@ -1,21 +1,55 @@
...
+}
 
-/* STRETCH PROBLEMS, Do not attempt until you have completed all previous tasks for today's project files */
+simpleCounter();
+simpleCounter();
 
+/* STRETCH PROBLEMS, Do not attempt until you have completed all previous tasks for today's project files */
 
 // ==== Challenge 2: Create a counter function ====
 const counter = () => {
```
Comment (GianBarGian) [inline assignments/closure.js:15]: Good job with stretch goals on closures!

File: assignments/callbacks.js (hunk 1 above)
Comment (daquinons) [reply assignments/callbacks.js:35]: @GianBarGian Fixed, I think :)

Comment (GianBarGian) [reply assignments/callbacks.js:35]: No, the way the functions was declared was fine. 
It's just that you are not trying to use it anywhere after is declared.  
In this way you are not trying to actually use callbacks functions. You could want to maybe write a logger function and pass it as the second argument of your functions.
If it is not clear how DM me.

Comment (daquinons) [reply assignments/callbacks.js:35]: Ooooh ok, I get it now! Fixed (now for real haha).

Comment (GianBarGian) [reply assignments/callbacks.js:35]: Yeah now it looks perfect. Well done!

**Comment by GianBarGian** [comment]
Good job with your project. Everything looks perfect now that you added your invocations to your callback file. 
Well done!

**Pull request merged**